service/commentutil/commentutil.go

```diff
@@ -77,6 +77,35 @@ func MarkdownComment(c *reviewdog.Comment) string {
 	return sb.String()
 }
 
+// MarkdownSuggestions creates diff in markdown for suggested changes
+func MarkdownSuggestions(c *reviewdog.Comment) string {
```

Comment:
This markdown syntax (suggestion block) is not common but it's specific to GitLab. Can you move this func to gitlab file?